.github/workflows/integration-test.yml

```diff
@@ -91,7 +91,7 @@ jobs:
       - name: Generate matrix
         id: set-matrix
         run: |
-          go run --tags=generator generator/test_case_generator.go
```

Comment:
why are you removing tags?

Reply:
The `--tags` flag in the go run command is used to specify build tags, but the `generator/test_case_generator.go` doesn't have any build tags.

Though, I can change it back to prevent touching this workflow.